### PR TITLE
🌷🌷Added Vitess Integration

### LINKED ISSUE #3682: [New Integration]: 😎😎 Integration With Vitness
### Is there an existing integration?

- [X] I have searched the existing integrations.

### Use Case

[Vitess ](https://vitess.io/)is a database clustering system for horizontal scaling of MySQL through generalized sharding.

By encapsulating shard-routing logic, Vitess allows application code and database queries to remain agnostic to the distribution of data onto multiple shards. With Vitess, you can even split and merge shards as your needs grow, with an atomic cutover step that takes only a few seconds.

### Motivation

Vitess can help you with the following problems:

  *  Scaling a SQL database by allowing you to shard it, while keeping application changes to a minimum.
  * Migrating from baremetal to a private or public cloud.
  * Deploying and managing a large number of SQL database instances.

### Implementation

_No response_

### Anything else?

If You Like THis Idea THen Please Give IT 👍

### PR DESCRIPTION
**Pull request by parthiv11**
Fixes #3682 

## Please describe what changes you made, in as much detail as possible
  - Added Vitess Integration Test it and suggest changes if required

mindsdb

### CODE REVIEW

**Comment by parthiv11** [comment]
For testing use  [docker ](https://vitess.io/docs/15.0/get-started/vttestserver-docker-image/) or [other options](https://vitess.io/docs/15.0/get-started/)

File: mindsdb/integrations/handlers/vitess_handler/__init__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+except Exception as e:
+    Handler = None
+    import_error = e
+from .__about__ import __version__ as version, __description__ as description
+
+
+title = 'Vitess'
+name = 'vitess'
+type = HANDLER_TYPE.DATA
+icon_path = 'icon.png'
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/vitess_handler/__init__.py:15]: Please include the icon in the PR

File: mindsdb/integrations/handlers/vitess_handler/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  *  Scaling a SQL database by allowing you to shard it, while keeping application changes to a minimum.
+  * Migrating from baremetal to a private or public cloud.
+  * Deploying and managing a large number of SQL database instances.
+
+## Implementation
+
+This handler was implemented by extending mysql connector.
+
+The required arguments to establish a connection are:
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/vitess_handler/README.md:16]: Isn't the keyspace and schema required?

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

**Comment by parthiv11** [comment]
I think it is working without that parameters

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**